asteroid/modules/prologue.ast

```diff
@@ -973,3 +973,21 @@ with none do
                 @reverse()
                 @join("").
 end
+
+------------------------------------------------------------------
+--Repl autocompletion toggle
+------------------------------------------------------------------
+escape
+"
+import os
+import sys
+ 
+file_path = os.path.dirname(os.path.abspath( __file__ ))
```

Comment:
I don't understand what's going on here? Why is this in the prologue file and not in your repl module?

If this is here in order for your code to find the lex module then your sandbox setup is incorrect.